src/mds/Server.cc

```diff
@@ -4452,6 +4452,26 @@ void Server::handle_client_openc(MDRequestRef& mdr)
     return;
   }
 
+  CF_MDS_RetryRequestFactory cf(mdcache, mdr, false);
+  int r = mdcache->path_traverse(mdr, cf, req->get_filepath(), 0, NULL);
+  if (r > 0) return;
+  if (r == 0) {
+    // it existed.
+    handle_client_open(mdr);
+    return;
+  }
+  if (r < 0 && r != -ENOENT) {
+    if (r == -ESTALE) {
```

Comment:
Please switch to `-CEPHFS_*`.

Reply:
Done.
I attempted to use the latest version, but I have some compilation problems one centos8. At present, this modification is mainly tested on version 15.